src/main/java/com/exmertec/dummie/DummyBuilderFactory.java

```diff
@@ -0,0 +1,29 @@
+package com.exmertec.dummie;
+
+
+import com.exmertec.dummie.cache.DummyCache;
+import com.exmertec.dummie.cache.impl.DefaultCache;
+import com.exmertec.dummie.cache.impl.LevelCache;
+
+public class DummyBuilderFactory {
```

Comment:
As discussed, no need another builder factory, should use "convention over configuration"